stdlib/2/subprocess.pyi

```diff
@@ -4,7 +4,7 @@
 
 from typing import Sequence, Any, Mapping, Callable, Tuple, IO, Union, Optional, List, Text
 
-_FILE = Union[int, IO[Any]]
+_FILE = Optional[Union[int, IO[Any]]]
```

Comment:
Yeah cool - I'm all for keeping it the same if it's done this way elsewhere. Cheers for the explanation.